src/lib/frontend.ts

```diff
@@ -43,6 +54,28 @@ export class ElectronFrontend extends FrontendBase<
     const prepared = await this.prepareEvent(event);
     await this.getBackend().uploadMinidump(path, prepared);
   }
+
+  /**
+   * @inheritDoc
+   */
+  public async addBreadcrumb(breadcrumb: Breadcrumb): Promise<void> {
+    if (isRenderProcess()) {
```

Comment:
We should do the same for `ElectronFrontend.captureEvent` and instead add an assert in `ElectronBackend.sendEvent`. I just realized that otherwise an event goes through the lifecycle hooks twice (once in the renderer and once in the main process).

Reply:
👍